bin/config.sh

```diff
@@ -148,25 +173,49 @@ clean_config()
     if [ "$1" == "amster" ]; then
         rm -rf "$DOCKER_ROOT/$1/config"
     elif [ "$1" == "am" ]; then
-	rm -rf "$DOCKER_ROOT/$1/config"
+	      rm -rf "$DOCKER_ROOT/$1/config"
     elif [ "$1" == "idm" ]; then
         rm -rf "$DOCKER_ROOT/$1/conf"
-	rm -rf "$DOCKER_ROOT/$1/script"
-	rm -rf "$DOCKER_ROOT/$1/ui"
+	      rm -rf "$DOCKER_ROOT/$1/script"
+	      rm -rf "$DOCKER_ROOT/$1/ui"
     elif [ "$1" == "ig" ]; then
         rm -rf "$DOCKER_ROOT/$1/config"
         rm -rf "$DOCKER_ROOT/$1/scripts"
-	rm -rf "$DOCKER_ROOT/$1/lib"
+	      rm -rf "$DOCKER_ROOT/$1/lib"
     fi
 }
 
-# Copy the product config $1 to the docker directory.
 init_config()
 {
-    if [ -d "${PROFILE_ROOT}/$1" ]; then
-        echo "cp -r ${PROFILE_ROOT}/$1" "$DOCKER_ROOT"
-        cp -r "${PROFILE_ROOT}/$1" "$DOCKER_ROOT"
+  if [ -d "${PROFILE_ROOT}/$1" ]; then
+    echo "*********************************************************************************************"
+    echo "Initialisation of 'docker/7.0/$1' for [$_arg_environment] environment in [$_arg_ig_mode] mode"
+    echo "*********************************************************************************************"
+    echo "copy ${PROFILE_ROOT}/$1/lib to $DOCKER_ROOT/$1/"
+    cp -r "${PROFILE_ROOT}/$1/lib" "$DOCKER_ROOT/$1/"
+    echo "copy ${PROFILE_ROOT}/$1/scripts to $DOCKER_ROOT/$1/"
+    cp -r "${PROFILE_ROOT}/$1/scripts" "$DOCKER_ROOT/$1/"
+    echo "copy ${PROFILE_ROOT}/$1/config/$_arg_environment/config to $DOCKER_ROOT/$1/"
+    cp -r "${PROFILE_ROOT}/$1/config/$_arg_environment/config" "$DOCKER_ROOT/$1/"
+    jq --arg mode "${_arg_ig_mode^^}" '.mode = $mode' "$DOCKER_ROOT/$1/config/"admin.json > "$DOCKER_ROOT/$1/config/"admin.json.tmp
+    mv "$DOCKER_ROOT/$1/config/"admin.json.tmp "$DOCKER_ROOT/$1/config/"admin.json
+    echo "IG mode $_arg_ig_mode"
+    if [ "$_arg_ig_mode" == "development" ]; then
```

Comment:
Depends of the `--igmode` argument the routes will live in a different folders in the Ig configuration:
- For development mode all routes will live in `${IG_HOME}/config/routes`
- For production mode all routes will live ni the same folder defined in `configuration master area`